apps/index.html

```diff
@@ -305,6 +305,22 @@
       </div>
 
       <div class="app">
+        <button onclick="install('118d77fa72eba6a685cb493bffb18fae?url=https://github.com/libreboard/libreboard/releases/download/v0.7.1/libreboard.spk')">Install »</button>
+        <h3>LibreBoard</h3>
+        <div class="app-details">
+          Developed by: <a href="https://github.com/yasaricli">Yaşar İçli</a> &amp; <a href="https://github.com/mquandalle">Maxime Quandalle</a><br>
+          License: MIT<br>
+          Code: <a href="https://github.com/libreboard/libreboard">On GitHub</a><br>
+          Updated: Jan 11 2015
```

Comment:
Personally I would also like to put a version number here; `Version: 0.7.1`, but this is not consistent with other applications. Should I do it anyway?